### PR TITLE
CHANGELOG: update changelog for the 20.7 release

### PR DESCRIPTION
**Pull request by amstewart**
Misc. whitespace changes and trivial formatting throughout.

Signed-off-by: Alex Stewart <alex.stewart@ni.com>

Will also be merged into the sumo and dunfell mainline changelists.

@ni/rtos

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,25 @@ This changelog attempts to conform to the changelog spec on [keepachangelog.org]
...
 Branch: `nilrt/master/sumo`
 
+
+## 8.7
+Branch: `nilrt/20.7/sumo`
+
+### meta-nilrt
+
+#### Added
+- [Added](https://github.com/ni/meta-nilrt/pull/63) an unsupported preview of newer kernel bases (currently 5.10) via the `kernel-next` recipe.
```
Comment (gratian) [inline CHANGELOG.md]: Should we mention the package group here instead of the recipe? i.e. `packagegroup-ni-next-kernel`
It was introduced a while ago by this commit (so it might not fit under 8.7):
- https://github.com/ni/meta-nilrt/commit/4e075099adf828d17fd9c111c043efdba55247f1
and then extended by the 'kernel-devsrc' you've mentioned:
- https://github.com/ni/meta-nilrt/commit/0ce447888ee09add1a8f83656b102e32d7af50a5

Basically instead of mentioning `via the kernel-next recipe` say `via the packagegroup-ni-next-kernel ...`
It would provide a better starting point for somebody wanting to try the upcoming kernel.

**Review by gratian — APPROVED** [review]
LGTM with the minor nit around using the packgegroup for the next kernel instead of talking about the recipe.

File: CHANGELOG.md (hunk 1 above)
Comment (amstewart) [reply CHANGELOG.md]: Makes sense; will fix.

**Comment by amstewart** [comment]
## Update 2
* Changed the `kernel-next` entries to refer to the packagegroup (and href to the packagegroup PRs), rather than the individual recipes.

**Review by shruthi-ravi — APPROVED** [review]
